src/main/java/frc/robot/subsystems/Drive/DriveSubsystem.java

```diff
@@ -317,4 +318,13 @@ public void periodic() {
      public void simulationPeriodic() {
           // This method will be called once per scheduler run during simulation
      }
+
+     @Override
+     public void onLimelightDataUpdate(boolean targetVisible, double[] botPose) {
+          
+          double x = botPose[0];
+          double y = botPose[1];
+          double rotationRadians = botPose[2];
+          currentPose = new Pose2d(x, y, new Rotation2d(rotationRadians));
```

Comment:
use a diffrent varaiable because there is allready an odemtry in the robot it will mess it up, so use a different variable insted of currentPose to store it